pages/[organizationName]/help.tsx

```diff
@@ -33,10 +33,28 @@ import { Organization } from "."
   function Help({ organization }: InferGetServerSidePropsType<typeof getServerSideProps>) {
     const classes = useStyles(organization)
 
+    // Responsive page
+    const [width, setWindowWidth] = useState(0)
+    useEffect(() => { 
+      updateDimensions();
+      window.addEventListener("resize", updateDimensions);
+      return () => 
+        window.removeEventListener("resize", updateDimensions);
+    }, [])
+
+    const updateDimensions = () => {
+      const width = window.innerWidth
+      setWindowWidth(width)
+    }
+
+    const screenWidth = {
+      isDesktop: width > 1023,
+    }
+
```

Comment:
This is really cool, I like how is the same logic in each page, I guess its something that could be potentially be refactored to be stored in one place and inherited by the pages in some way

Reply:
Hi @Jnegrier, i wonder how, since it's a constant that gets updated every time the screen size changes. I will look into it